org.adempiere.ui/src/org/compiere/print/DrillReportCtl.java

```diff
@@ -180,7 +184,7 @@ private void initProcessDrillRuleMap() {
 			MProcessDrillRule[] processDrillRules = MProcessDrillRule.getByColumnName(Env.getCtx(), m_ColumnName, null);
 			for( MProcessDrillRule drillProcesRule: processDrillRules) {
 				MProcess process = MProcess.get(drillProcesRule.getAD_Process_ID());
```

Comment:
For better efficiency, checking should happens before `MProcess process = MProcess.get`. i.e `if (drillProcesRule.getAD_Process_ID() == 0 || drillProcesRule.getAD_Process_ID() == m_AD_Process_ID) continue;` (process can only be null if `drillProcesRule.getAD_Process_ID() == 0`).